features/providers/non_means_tested_journey.feature

```diff
@@ -0,0 +1,225 @@
+Feature: Non-means-tested applicant journeys
```

Comment:
Could this be split into a `non_means_tested_journey` folder with each scenario in a single feature file?

Reply:
@colinbruce I have split these up in a single commit. happy to go with this and discuss extending to other "journeys" at next dev time (trello discussion item added)